### PR TITLE
Moved apt-gets from .sh file to dockerfile for nanopb

### LINKED ISSUE #5781: Nanopb sanity test failed
We should move the apt-get install commands to the sanity dockerfile.

CC @jtattermusch 

https://grpc-testing.appspot.com/job/gRPC_master/15849/config=dbg,language=sanity,platform=linux/testReport/junit/(root)/tests/tools_distrib_check_nanopb_output_sh/

### PR DESCRIPTION
**Pull request by dgquintas**
Fixes #5781

### CODE REVIEW

File: templates/tools/dockerfile/test/sanity/Dockerfile.template [hunk 1]
```diff
@@ -34,7 +34,13 @@
   <%include file="../../apt_get_basic.include"/>
   #========================
   # Sanity test dependencies
-  RUN apt-get update && apt-get install -y python-pip
+  RUN apt-get update && apt-get install -y \
```
Comment (ctiller) [inline templates/tools/dockerfile/test/sanity/Dockerfile.template]: replace \ with ${"\\"} to preserve formatting in the generated file

**Comment by dgquintas** [comment]
Done.

**Comment by dgquintas** [comment]
Green-ish

**Comment by jtattermusch** [comment]
I've prebuilt the dockerimage on all  workers:
https://grpc-testing.appspot.com/view/All/job/gRPC_prebuild_dockerfiles/33/

Besides that, LGTM. Waiting for tests.

**Pull request merged**